tools/packager/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@
 cmake_minimum_required(VERSION 2.8.12)
 include(packaging)
 
+####### check python dependency ##############################################
+include(FindPythonPackage)
+FIND_PYTHON_PKG("jsonschema" SEND_ERROR)
+FIND_PYTHON_PKG("repoze" SEND_ERROR)
```

Comment:
Nice!

I just checked the DCP environment. Of course they have jsonschema, since they need it. I don't see repoze installed. Perhaps that should be a warning until you get them to install it? Otherwise, you'll break all their regressions.

Where is repoze used?

Reply:
Yeah, I don't think we have any dependency on repoze.

Reply:
latest commit removed the python repoze dependency check.